index.js

```diff
@@ -101,9 +102,8 @@ function commit (argv, newVersion, cb) {
     args.unshift('package.json')
   }
   checkpoint(argv, msg, args)
-
```

Comment:
OK I readded the whitespace, not sure why but it's not showing up in the diff here.